dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryContainerFilter.java

```diff
@@ -124,6 +139,8 @@ public void filter(final ContainerRequestContext request, final ContainerRespons
 
         try {
             instrumenter.end(spanContext, request, response, null);
+        } catch (Exception e) {
+            Tr.error(tc, Tr.formatMessage(tc, "CEMOT5101.telemetry.error", e));
         } finally {
```

Comment:
We're more worried about unexpected exceptions from our own code than we are about unexpected exceptions from OTel. This try-catch should go around the whole method.

Same for any other methods where you do this, the try-catch should be around the whole method.